### PR TITLE
Editor: Fixed non-localized history entries

### PR DESCRIPTION
**Pull request by ycw**
This PR extracted and translated (inaccurately, via online translators) all commands' `name` which are shown as history entries:

ja

Preview: https://raw.githack.com/ycw/three.js/editor-localize-commands/editor/index.html

**Translation contributors may improve them _after_ this PR merged**, thank you.

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

File: editor/js/Sidebar.Material.js [hunk 1]
```diff
@@ -500,7 +500,7 @@ function SidebarMaterial( editor ) {
 
 				}
 
-				editor.execute( new SetMaterialCommand( editor, currentObject, material, currentMaterialSlot ), 'New Material: ' + materialClass.getValue() );
+				editor.execute( new SetMaterialCommand( editor, currentObject, material, currentMaterialSlot ), strings.getKey( 'command/SetMaterial' ) + ':' + materialClass.getValue() );
```
Comment (Mugen87) [inline editor/js/Sidebar.Material.js:503]: For readability, it would be good to retain the space after the `:` sign.

Comment (ycw) [reply editor/js/Sidebar.Material.js:503]: `xxx : yyy` or `xxx: yyy` ?

Comment (Mugen87) [reply editor/js/Sidebar.Material.js:503]: `xxx: yyy` as it was before.

Comment (ycw) [reply editor/js/Sidebar.Material.js:503]: ok.

**Review by Mugen87 — APPROVED** [review]

**Pull request merged**